src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1003,6 +1003,7 @@ public static function redirect($strLocation, $intStatus=303)
 			exit;
 		}
 
+		$strLocation = ltrim($strLocation, '/');
```

Comment:
Because `$request->getRequestUri()` returns the URI including the leading `/` which is correct. Our `Controller::redirect()` method would turn things into something like that: `http://domain.de//path/to/whatever`. I thought it's better to ensure there's never a leading `/` than doing stupid `str_replace()` calls **everytime** we pass something into that legacy method.